library/src/main/java/com/mapzen/android/MapView.java

```diff
@@ -126,6 +126,28 @@ public void getMapAsync(@NonNull String key,
     }
 
     /**
+     * Load map asynchronously using given API key.
+     *
+     * @param callback listener to be invoked when map is initialized and ready to use.
+     * @param key vector tiles API key that should be used to load map tiles.
+     * @param sceneFile scene file that should be used
+     */
+    public void getMapAsync(@NonNull String key,
+            @NonNull String sceneFile, @NonNull OnMapReadyCallback callback) {
+        mapInitializer.init(this, key, sceneFile, callback);
+    }
+
+    /**
+     * Load map asynchronously using given API key.
+     *
+     * @param callback listener to be invoked when map is initialized and ready to use.
+     * @param sceneFile scene file that should be used
+     */
+    public void getMapAsync(@NonNull OnMapReadyCallback callback, @NonNull String sceneFile) {
```

Comment:
We should probably switch the order of the params and make `callback` last here to keep consistent with the other versions of this method.